test/testvm.py

```diff
@@ -23,8 +23,6 @@
 import json
 import libvirt
 import libvirt_qemu
-from   lxml import etree
```

Comment:
I see you added etree, ok

https://github.com/cockpit-project/cockpit/blob/master/test/testvm.py#L611